tests/fixtures/test-project/expected-with-call-graph.json

```diff
@@ -0,0 +1,48 @@
+{
+  "plugin": { "name": "bundled:maven", "runtime": "unknown" },
+  "callGraph": {},
```

Comment:
Will there be a test with actual or semi-actual call graph?

Reply:
It's pretty heavy to test to full functionality of the call graph builder, I can maybe add a more useful graph just for the fixtures.

Reply:
A full integration test will be part of the CLI